src/commands/time/finish.js

```diff
@@ -26,102 +26,106 @@ function controller(t) {
         var isTask = !isBug && e.tp_task;
 
         utils.log('    Logging time on target process...');
-        if(isBug) {
-          utils.log('    Bugs are configured to be logged against: ' + tp.bugTimeBehavior);
-        }
+        tp.getBug(e.tp_task.id).then(function (result) {
```

Comment:
Is this not going to error if the TP entity we are trying to log time to is not a bug? e.g. it is a User Story or Task?
Could we do this after checking isBug first?

Reply:
If we check what isBug is first, how do we get the custom field value?

Reply:
We only ever need to get the value of the custom field if the TP entity is a bug, since its only with bugs where we would use different behaviours.

Then if it is a bug, we can go ahead and do what you're doing already.